### ISSUE TITLE
[homepage] Scrolling on the extreme left scrolls the right column

### ISSUE THREAD

**Issue by boogheta** [medialab/website#471]
I guess the page is essentially the right column, with a scrollable div on top containing the left column, but the left column isn't fixed on the left border of the page resulting in this strange behavior, maybe it could be possible to simple make the div larger with some internal padding

**Comment by Yomguithereal** [comment]
@robindemourat what’s your expert opinion on this :)?

**Comment by robindemourat** [comment]
yes it seems weird (but you have to look for it ;)), I can give it a look next week to see if there's a quick fix !

**Comment by BenjmnG** [comment]
Yes, it's logically weird.
The `section id="home"` (the left one) is fixed , height limited and
scrollable.
`section id="flux"` is relative to the page. So you scroll page, you scroll
the Flux

We could inverse the logic but in this case, we'll inverse the problem 🙃
You scroll right side, you scroll home section.

May we do the two sections fixed and scrollable but we we'll have, not one,
but two scrollbar on the center of screen.


Le mar. 21 janv. 2020 à 14:00, Robin de Mourat <notifications@github.com> a
écrit :

**Comment by boogheta** [comment]
A naive attempt with simple css does part of the trick:
```
.page-home #home {
  left: 0;
  padding-left: 30px;
}
```

Of course it's too simple and does not work well for mobiles and on big screens it breaks the alignment with the logo in the header but I'm sure there is a solution to fix this ! :)
